src/layout/css/style.scss

```diff
@@ -1,23 +1,33 @@
+$font-size-text: 12px;
+$font-size-h1: 2*$font-size-text;
```

Comment:
Can we not describe `$font-size-h1` and `$font-size-h2` in terns of `$font-size-text`? That makes it look like there's a reason behind the relation, but I don't think that's the case. It would also make things easier to read ( at least I have an easier time reading `16px` v. `round(1.3 * $font-size-text)`

Reply:
done